### PR TITLE
Add zabbix output plugin

### PR DESCRIPTION
**Pull request by UnderGreen**
Added output plugin, which sends metrics to Zabbix Server.

It's my first Golang code, please review it in depth. :smile:

### CODE REVIEW

**Comment by zstyblik** [comment]
@UnderGreen my comments are out of curiosity and not a code review. Thanks. :)

File: outputs/zabbix/zabbix.go [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+	if _, err := connection.Write(buffer); err != nil {
+		return fmt.Errorf("Zabbix: Sender writing error %s", err.Error())
+	}
+
+	return nil
+}
+
+func buildValue(pt *client.Point) (string, error) {
+	var retv string
+	var v = pt.Fields()["value"]
```
Comment (sparrc) [inline outputs/zabbix/zabbix.go]: Could you change this to loop through `pt.Fields()`? Telegraf is soon not going to only have a single `value` field in each measurement

Comment (sparrc) [inline outputs/zabbix/zabbix.go]: see the 0.3.0 version of the OpenTSDB plugin: https://github.com/influxdata/telegraf/blob/0.3.0/outputs/opentsdb/opentsdb.go

**Comment by sparrc** [comment]
Thank you @UnderGreen! If you could modify it to fit better with the upcoming 0.3.0 release, I would appreciate it a lot (see my comments)

**Comment by sparrc** [comment]
@zstyblik Try making your comments again on the lines of code, they are currently just in the general comments and out-of-context

**Comment by sparrc** [comment]
closing this pull request for lack of updates. @UnderGreen I'll review if you put up a new rebased PR, thanks!

**Comment by zstyblik** [comment]
@sparrc didn't I make comments against line numbers? If I click at https://github.com/UnderGreen/telegraf/commit/1d962bc2be611129e9df389958f8a19c12dfdde5 then I see my comments below lines that I have been concerned with/had questions to?
But indeed, my comments ended up here in general discussion :\

**Comment by sparrc** [comment]
@zstyblik you should comment on the "files changed" section of the PR, not directly on the commits

**Comment by zstyblik** [comment]
@sparrc I see. Hopefully, will do next time.

**Comment by frankgerhardt** [comment]
> closing this pull request for lack of updates. @UnderGreen I'll review if you put up a new rebased PR, thanks!

I'm looking at this and it's years old. My question is: is there Zabbix support by now and if yes where?

**Comment by Hipska** [comment]
@frankgerhardt It will be in v1.30.0: #13739

**Pull request closed without merging**